_posts/en/posts/2016-10-28-segwit-costs.md

```diff
@@ -0,0 +1,621 @@
+ 5. allowing large blockchain forks, due to the p2p network being
+    effectively disconnected as a result of connections via old nodes
+    that are unable to forward witness data
+
+## Avoidance
+
+Numerous soft-forks have already been activated in Bitcoin (including BIPS
+16, 34, 65, 66, 68, 112, 113), and this experience has been codified in the
+[BIP 9](https://github.com/bitcoin/bips/blob/master/bip-0009.mediawiki)
+process for activating soft-forks. The BIP 9 process was used for
+deploying the CSV soft-fork (BIPs 68, 112, and 113), and resulted in a
+fast and unproblematic upgrade to the consensus rules for that change.
+
+The segwit design and BIP 9 deployment avoids the problems listed above
+in the following ways:
+
+ 1. The new restrictions imposed by segwit only affect transactions
+    that no one would currently make use of:
+
+    - The affected transactions would be non-standard, and thus not
+      relayed by the vast majority of nodes or mined by most miners.
+
+    - Any transactions that were affected would currently be considered
+      obvious "anyone can spend" payments, and could immediately be
+      claimed by anyone monitoring the blockchain, and therefore should
+      have been expected to be "lost".
+
+ 2. Old nodes will consider transactions spending segwit transactions as
+    non-standard, due to apparent violation of BIP61 CLEANSTACK rules,
+    and thus won't be included in old nodes' mempools. Similarly, transactions
+    with P2WPKH or P2WSH outputs (though not P2WPKH/P2WSH encoded via P2SH)
+    will also be considered non-standard due to being a new output type.
+
+    This makes it impossible to achieve double spends of segwit outputs
+    by relaying one transaction through old nodes and a different
+    transaction through segwit nodes.
+
+    However, these differences may still be used to attempt a double
+    spend, for example by combining a non-segwit output and a segwit
+    output in a single transaction (that will only be relayed via the
+    upgraded segwit nodes), then attempting to double-spend it via
+    a higher fee transaction only using the non-segwit output, which may
+    be successful relayed via the old nodes.
+
+    These concerns only affect unconfirmed transactions in the mempool;
+    once a transaction is confirmed and mined in a block, double spending
+    remains impossible. Existing methods for monitoring double spends
+    should remain equally effective, provided the monitoring tools are
+    able to track segwit spends at all.
+
+ 3. Ensuring miners mine valid blocks is obviously a high priority to
+    everyone involved, and significant work has
+    gone into guaranteeing this is the case with
+    segwit. This includes both the direct work, under
+    [BIP-145](https://github.com/bitcoin/bips/blob/master/bip-0145.mediawiki),
+    as well as indirect work, such as Compact Blocks
+    ([BIP-152](https://github.com/bitcoin/bips/blob/master/bip-0145.mediawiki)).
```

Comment:
Note, this and a two other mislink errors is being corrected on my branch by using `_references.md` with reference style links for all BIP links that don't link to a specific document section.  (Just mentioning this here in case anyone sees them and so @ajtowns doesn't need to bother fixing them for the moment.)